### PR TITLE
Added setup instructions to main documentation for light node tutorials

### PR DESCRIPTION
**Pull request by dabit3**

### CODE REVIEW

**Comment by YazzyYaz** [comment]
@dabit3 looks like it fails the lint checks, can you resolve those.

File: docs/developers/node-tutorial.md [hunk 1]
```diff
@@ -4,49 +4,106 @@
...
+
+- Memory: 2 GB RAM
+- CPU: Single Core
+- Disk: 5 GB SSD Storage
+- Bandwidth: 56 Kbps for Download/56 Kbps for Upload
 
 ## Setting Up Dependencies
 
-You can follow the tutorial for setting up the dependencies [here](./environment.md).
+First, make sure to update and upgrade the OS:
```
Comment (YazzyYaz) [inline docs/developers/node-tutorial.md:22]: This means they're still building on Ubuntu btw

File: docs/developers/node-tutorial.md [hunk 2]
```diff
@@ -57,14 +114,45 @@ an example public Core Endpoint.
   and there are several in the Discord. This one is used for demonstration
   purposes. You can find a list of RPC endpoints [here](../nodes/mamaki-testnet#rpc-endpoints)
 
-Here we are starting a light node with a connection to a Core endpoint and
-also telling the light node to use the `developer` key we generated
-as its default account.
+```sh
+celestia light start --core.grpc http://<ip-address>:9090
```
Comment (YazzyYaz) [inline docs/developers/node-tutorial.md:118]: I would wait on this as we have a new PR that updates the CLI endpoint

File: docs/developers/node-tutorial.md (hunk 1 above)
Comment (dabit3) [reply docs/developers/node-tutorial.md:22]: Should I comment this somewhere? 

We mention both YUM and APT package managers, should we elaborate?

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**